### ISSUE TITLE
Add an ability to store the content of the fields separately

### ISSUE THREAD

**Issue by Stvad** [Stvad/CrowdAnki#33]
Possibly some other parts of the deck as well.

The inspiration is https://github.com/OnkelTem/anki-dm

Use cases:
* Localization (see #31  https://github.com/axelboc/anki-ultimate-geography)
* Customizations (i.e. slight changes for some fields)
    * for this it'd be useful to support some form of hierarchy (i.e. some Base values + override on top). Actually can be useful for localization use-case as well

**Comment by aleksejrs** [comment]
Hi, is the above issue (https://github.com/evolverine/covid-19-anki-deck/issues/6) a result of what this one is about?

**Comment by aplaice** [comment]
I don't think it's *really* what this issue is about.

AFAICT (based on the comparison with `anki-dm` of which I'm a happy-ish user (as part of `anki-ultimate-geography`), though I'd be happier if this functionality were part of `CrowdAnki` itself (and yes, I've been thinking about trying to implement this)) this issue is about splitting out the contents of the fields into, (probably) a separate file, where they could be stored in an arguably more readable format, say CSV (rather than as a list of JSON objects, as is the case now).

AFAICT your problem is that you add a field to all the notes in the upstream deck and when you update, your newly added field is removed (since it's not present upstream).

What I think might help you most would be #62, which would allow for some fields to be ignored on import/export. 

Depending on exactly how #62 is implemented (I don't remember what exactly is the current consensus in that thread and the linked PR), if say "method 1." is used, it might allow you to simply add several lines to the "flds" object in the relevant note model(s), along the lines of:

```json
        {
          "name": "MorphMan_FocusMorph",
           [...]
          "doNotImport": true
        }
```

You'd probably need to keep this tiny patch rebased on top of the latest master, but since most of the updates would be in the content, further down in the file, it shouldn't be particularly hard. (What this issue (33) might help with, is having all of the updates to the content happening in a separate file (or files), hence allowing for clean rebases.)

(If yet another, not currently suggested, approach is used, allowing for each end-user to set ignorable fields (externally to `deck.json`) then you wouldn't even need to keep rebasing such a patch.)

**Comment by ohare93** [comment]
I agree with @aplaice, this is not relevant to your issue @aleksejrs. I commented on that issue with my thoughts, as I am also a user of Morphman :+1: Let's talk there, to keep this issue clean.

Totally agreed with your thoughts on #62 @aplaice. I have a dirty working version of "Personal Fields" being ignored on import in #72, which I hope to expand in the coming weeks. Happy to discuss how it *should* work in the meantime, especially if you also want to take a shot at it! :grin:
